components/sysutils/nut/Makefile

```diff
@@ -101,27 +134,85 @@ CONFIGURE_OPTIONS +=	--with-net-snmp-config=$(USRBINDIR)/net-snmp-config-$(BITS)
 # Note: building the numerous documents of this projects is time-consuming
 # and we only need one copy. Also, we don't ship "dblatex" (needed here for
 # compiling PDFs) and probably don't require "html-chunked" chapters.
-CONFIGURE_OPTIONS +=        --with-doc="man=yes"
+CONFIGURE_OPTIONS +=	--with-doc="man=yes"
+
+# NUT-Monitor GUI client
+CONFIGURE_OPTIONS +=	--with-nut-monitor=yes
+CONFIGURE_OPTIONS +=	--with-pynut=yes
+CONFIGURE_OPTIONS +=	--with-python2=no
+CONFIGURE_OPTIONS +=	--with-python=python$(PYTHON_VERSION)
+
+# Be sure package is squeaky clean
+CONFIGURE_OPTIONS +=	--enable-Werror --enable-warnings --enable-Wcolor
+CONFIGURE_OPTIONS +=	--enable-keep_nut_report_feature
+CONFIGURE_OPTIONS +=	--enable-silent-rules
+
+# What do we run in case of `make check`?
+CONFIGURE_OPTIONS +=	--enable-spellcheck
+# NUT Integration Test suite, can take time and might fail due
+# to environment (busy ports, congested hsots and timeouts):
+#CONFIGURE_OPTIONS +=	--enable-check-NIT
+# No CPPCHECK here yet:
+CONFIGURE_OPTIONS +=	--disable-cppcheck
 
 CONFIGURE_OPTIONS +=	$(CONFIGURE_OPTIONS.$(BITS))
 
+# Fiddle with packaging:
+# * move man1m pages to target directory (see also patches/07-man.patch)
+# * move python stuff to vendor-packages
```

Comment:
Why is this needed?

Reply:
Hm, `site-packages` vs. `vendor-packages`.  There are probably better ways how to do it, but okay.

Reply:
Old recipe versions (by Alex P., I believe) mentioned a failure to install `*.1m` man pages into `man1m`, such comment was in the patch file as well.

Move from site-packages to vendor-packages (and pre-compilation) was picked up from a few existing components, probably `meld`, as I revised existing best practices about python package installations. There are a few hits to `git grep vendor-packages | grep -iw mv`. If "vendor" means OS distro, this makes sense...